src/Modules/Backend/assets/Backend/webpack/js/Backend.js

```diff
@@ -70,6 +72,12 @@ export class Backend {
       element.passwordStrengthMeter = new PasswordStrenghtMeter($(element))
     })
   }
+
+  static initTogglePasswordInputType () {
+    $('[data-role="toggle-visibility"]').each((index, element) => {
```

Comment:
Isn't `toggle-visibility` a bit too broad? I would use `toggle-password` to be inline with all other naming

Reply:
Fixed a0c40abde